client/src/components/exercise/lab11/pages/RepairImpls/SentenceCountRepairImplementation.js

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable react/prop-types */
+import ReactText from "../../../../all-components/CodeBlock/StyleComponents/ReactText";
+import React, { Fragment } from "react";
+import PropTypes from "prop-types";
+import CodeLine from "../../../../all-components/CodeBlock/Components/CodeLine";
+import Tab from "../../../../all-components/CodeBlock/Components/Tab";
+import CodeBlockInput from "../../../../all-components/CodeBlock/Components/CodeBlockInput";
+import CommentText from "../../../../all-components/CodeBlock/StyleComponents/CommentText";
+
+const SentenceCountRepairImplementation = (props = {}) => {
+  const { fogIndexCalculationData, userInput } = props;
+
+  return (
+    <>
```

Comment:
rather than encapsulating all code block data inside a fragment, can we create a parent component that takes code block components? something like `CodeBlockParent`

Reply:
If im not mistaken thats what Repair.js is